java/org/apache/catalina/valves/AccessLogValve.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.juli.logging.LogFactory;
 import org.apache.tomcat.util.ExceptionUtils;
 import org.apache.tomcat.util.buf.B2CConverter;
+import org.apache.tomcat.util.file.CompressFileUtils;
 
```

Comment:
Why not use the JULI version since the dependency is already there rather than duplicating the code.